src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -394,12 +474,25 @@ function enhancedCommands(commands, types)
 
 function enhancedAttributes(attributes, types)
 {
+  const kGlobalAttributes = [ 0xfffc, 0xfffd ];
+
   attributes.forEach(attribute => {
     enhancedItem(attribute, types);
+    attribute.isGlobalAttribute     = kGlobalAttributes.includes(attribute.code);
+    attribute.isWritableAttribute   = attribute.isWritable == 1;
```

Comment:
should we prefer using === instead of == ? or could our types have here strings or other things?

Reply:
There are not supposed to be strings. `===` should work.